owncloudComLibrary/build.gradle

```diff
@@ -18,11 +18,11 @@ dependencies {
 }
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 30
```

Comment:
We won't update to SDK 30 yet. We need to research and probably apply some changes regarding the scoped storage. We will update the SDK, but not in this PR. Could you drop this commit?